### PR TITLE
add gha for cli based sandbox

### PR DESCRIPTION
**Pull request by scott-cotton**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -145,3 +145,49 @@ jobs:
...
+            -e SIGNADOT_ORG=${SIGNADOT_ORG} \
+            -e SIGNADOT_API_KEY=${SIGNADOT_API_KEY} \
+            signadot/signadot-cli \
+            /signadot sandbox apply \
+            --set image=signadot/hotrod:${IMAGE_TAG} \
+            --set git-short-commit=${IMAGE_TAG:0:6} \
+            -f - \
+            -o yaml \
+            < ${GITHUB_WORKSPACE}/.signadot/frontend-template.yaml \
+            > ${GITHUB_WORKSPACE}/.signadot/frontend.yaml
```
Comment (foxish) [inline .github/workflows/build.yml:176]: can we name this something that indicates that it's the output? It wasn't super clear at first pass.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -145,3 +145,49 @@ jobs:
...
+      - name: Delete Sandbox
+        env:
+          SIGNADOT_ORG: signadot
+          SIGNADOT_API_KEY: ${{ secrets.SIGNADOT_API_KEY }}
+        run: |
+          docker run -i \
+            -e SIGNADOT_ORG=${SIGNADOT_ORG} \
+            -e SIGNADOT_API_KEY=${SIGNADOT_API_KEY} \
+            signadot/signadot-cli \
+            /signadot sandbox apply \
```
Comment (foxish) [inline .github/workflows/build.yml:191]: `delete -f`? or just `delete`?

Comment (scott-cotton) [reply .github/workflows/build.yml]: oops, thx for the catch.

File: .github/workflows/build.yml (hunk 1 above)
Comment (scott-cotton) [reply .github/workflows/build.yml:176]: Trying to think of some alternatives: `frontend-sandbox.yaml`, `frontend-applied.yaml`, `frontend-output.yaml`.  

(Without an extension, this also plays off the description at https://deploy-preview-57--signadot-docs.netlify.app/docs/ci-integrations, which might make plain `frontend.yaml` easier to digest.)

wdyt?

**Review by foxish — APPROVED** [review]

**Pull request merged**